dpgen/generator/run.py

```diff
@@ -798,17 +822,26 @@ def run_train(iter_index, jdata, mdata):
             os.path.join("old", "model.ckpt.data-00000-of-00001"),
         ]
```

Comment:
The file `model.ckpt.pt` is collected, but not being used. I guess it should be used around here. However, I am not familiar with this process then I can not  make a change. Hope you check this.